### PR TITLE
Added AWS Marketplace Installation

### PR DESCRIPTION
**Pull request by muratkars**
- Added OpenEBS container 1.0.0 images hosted in Amazon ECR registry for AWS Marketplace listing

Signed-off-by: Murat Karslioglu <murat@mayadata.io>

### CODE REVIEW

**Comment by muratkars** [comment]
I have tested it on AWS.

You need to subscribe to the AWS Marketplace listing  https://aws.amazon.com/marketplace/pp/B07TFS9Q8D, follow the instructions to retrieve the login command to authenticate your Docker client to your registry and docker login to the registry before you can pull the images.

File: docs/openebs-operator-1.0.0-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2702475830/quay.io/openebs/m-exporter:1.0-latest"
+        - name: OPENEBS_IO_CSTOR_POOL_EXPORTER_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2702475830/quay.io/openebs/m-exporter:1.0-latest"
+        # OPENEBS_IO_ENABLE_ANALYTICS if set to true sends anonymous usage
+        # events to Google Analytics
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (kmova) [inline docs/openebs-operator-1.0.0-aws.yaml]: Can the value be changed to - `openebs-aws` ?

File: docs/openebs-operator-1.0.0-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,665 @@
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
+# NOTE: On GKE, deploy the openebs-operator.yaml in admin context
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.0.0-aws.yaml]: This line is not needed.

File: docs/openebs-operator-1.0.0-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_HELPER_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2702475830/quay.io/openebs/openebs-tools:1.0-latest"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.0.0-aws.yaml]: This value should be changed to `openebs-aws`

File: docs/openebs-operator-1.0.0-aws.yaml (hunk 1 above)
Comment (muratkars) [reply docs/openebs-operator-1.0.0-aws.yaml]: updated @kmova

File: docs/openebs-operator-1.0.0-aws.yaml (hunk 2 above)
Comment (muratkars) [reply docs/openebs-operator-1.0.0-aws.yaml]: Removed the note @ChandanSagar Thanks

File: docs/openebs-operator-1.0.0-aws.yaml (hunk 3 above)
Comment (muratkars) [reply docs/openebs-operator-1.0.0-aws.yaml]: Changed the value @ChandanSagar

**Pull request merged**